config/standard/contact-summary.templated.js

```diff
@@ -1,16 +1,30 @@
+var now = extras.now;
```

Comment:
is `extras` injected by medic-conf?

Reply:
For now, yes. Eventually, I'm hoping to replace it with `const { blah } = require('./nools-extras);`. This is my baby step toward that.